app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,11 @@
+class MovieActorsController < ApplicationController
+  def create
+    MovieActor.create!(movie_actor_params)
+    redirect_to "/movies/#{params[:movie_id]}"
```

Comment:
Keep in mind that if you don't seed the development database with anything, you won't be able to find any movies or actors by their id because the database is empty. You will set up test data in your tests so they can pass, but running your Rails server will probably lead to an error because there are no movies to visit show pages for.